src/main/java/stonks/ui/Ui.java

```diff
@@ -0,0 +1,58 @@
+package stonks.ui;
+
+import java.util.Scanner;
+import stonks.task.TaskManager;
+
+/**
+ * Deals with interactions with the user
+ */
+public class Ui {
+    /**
+     * Prints message given with separating lines
+     * @param message text to be displayed
+     */
+    public void showMessage(String message) {
+        String SEPARATOR = "    ____________________________________________________________\n";
+        System.out.print(SEPARATOR + message + "\n" + SEPARATOR);
+    }
+
+    /**
+     * Greets user when chatbot is opened
+     */
+    public void showGreeting() {
+        String greeting = "     Hello! I'm Stonks\n" +
```

Comment:
Could all these spaces be saved as a const instead?
```suggestion
private String NEW_LINE_INDENT = "     ";
...
String greeting = NEW_LINE_INDENT + "Hello! I'm Stonks\n" + NEW_LINE_INDENT + "What can I do for you?";